helmfile.d/charts/tekton-pipelines/templates/tekton-alerts.yaml

```diff
@@ -23,3 +23,21 @@ spec:
         increase(tekton_pipelines_controller_pipelinerun_count{status="failed"}[12h]) > 0
       labels:
         severity: critical
+  - name: tekton-task-slow
+    rules:
+    - alert: TektonNonUpgradeApplyTaskSlow
+      annotations:
+        description: Non upgrade task {{`{{ $labels.pod }}`}} has been running for over 5 minutes
+      expr: |
+        kube_pod_container_status_running{namespace="tekton-pipelines",pod=~".*alert-silence.*|.*clone-readme.*|.*clone-config.*|.*git-change.*|.*fetch-version.*|.*verify-k8s-api.*|.*verify-secret-encryption.*"} == 1
+      for: 5m
+      labels:
+        severity: warning
+    - alert: TektonUpgradeApplyTaskSlow
+      annotations:
+        description: Upgrade task {{`{{ $labels.pod }}`}} has been running for over {{ .Values.alerts.upgradeAppsAlertThreshold }}
+      expr: |
+        kube_pod_container_status_running{namespace="tekton-pipelines",pod=~".*upgrade-apps-pod.*"} == 1
+      for: {{ .Values.alerts.upgradeAppsAlertThreshold }}
+      labels:
+        severity: warning
```

Comment:
Feels like this should be added together with the pipeline in devbox.

Reply:
Yeah that makes more sense, I'll move this to a PR there instead. I guess I can leave this PR up for just adding the schema for the other values instead.